docs/guides/create-interface.md

```diff
@@ -0,0 +1,62 @@
+---
+id: create-interface
+title:  Create an Interface
+---
+
+## **Introduction**
```

Comment:
Would be cool to have more information about:
- what an interface is
- why you would want to use one (what use-cases would be good for using an interface)